LICENSE.BINARY

```diff
@@ -272,7 +272,7 @@ BINARY/WEB-CONSOLE
 
     This product bundles diff-match-patch version 1.0.4.
 
-    This product bundles @blueprintjs/core version 1.0.1.
+    This product bundles @blueprintjs/core and @blueprintjs/icons of version 3.15.0.
```

Comment:
Are these dependency lists and version numbers being updated automatically or are you doing it by hand? If it's by hand, let's think about how to automate it, since this seems like a lot of work!!

By the way, I think to make release managers' lives easier, anything that can't feasibly automated we should try to get done in the PR that makes the original changes. So these changes could have been done in #7398.

Reply:
Some part of the process is automated (`docs/_bin/npm-license-helper.py` and `docs/_bin/web-console-dep-lister.py`), but it still needs quite a lot of manual work. I'm thinking something to automate most of the part. Will raise an issue (or maybe proposal?) soon.